pkg/lib/cockpit-components-table.scss

```diff
@@ -106,3 +106,28 @@
         border-bottom: 1px solid var(--ct-color-list-critical-border);
     }
 }
+
```

Comment:
This might then go to some generic css file like page.css or so and then we can add the `.new-item` logic to other components and it would "just work" :tm: